packages/office-ui-fabric-react/src/components/TextField/examples/TextField.Examples.scss

```diff
@@ -1,5 +1,22 @@
+@import '../../../common/_variables';
+
 :global {
-  .ms-TextFieldExample .ms-TextField {
-    margin: 0 0 15px 0;
+  .ms-TextFieldExample {
+    .ms-TextField {
+      margin: 0 0 15px 0;
+      max-width: $maxFieldWidth;
+    }
+  }
+
+  .ms-TextFieldErrorExample {
+    .ms-TextField,
+    .NumberTextField {
+      margin: 0 0 15px 0;
+      max-width: $maxFieldWidth*2;
```

Comment:
Why does the error example need to be twice as wide? Could include a comment here, as it seems odd although I'm sure there's a reason.

Reply:
The validation descriptions were so long! Figured it'd be better user experience to have those visible. Double is somewhat arbitrary but seemed to work. I will definitely add a comment so that's clear.